bedrock/settings/static_media.py

```diff
@@ -1027,6 +1027,7 @@
             'js/newsletter/form.js',
             'js/base/mozilla-client.js',
             'js/base/mozilla-image-helper.js',
+            'js/base/mozilla-cookie-helper.js',
```

Comment:
As Traffic Cop runs before the `common` bundle is loaded, I think we need to add `mozilla-cookie-helper` to the `site` bundle.